etc/kayobe/inventory/group_vars/wazuh-manager/wazuh-manager

```diff
@@ -0,0 +1,155 @@
+---
+# Wazuh deployment details and documetation:
+# https://documentation.wazuh.com/current/deployment-options/deploying-with-ansible/index.html
+#
+# Defaults that are modified by this file:
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/ansible-filebeat-oss/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/ansible-wazuh-manager/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/wazuh-dashboard/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/wazuh-indexer/defaults/main.yml
+# https://github.com/wazuh/wazuh-ansible/blob/4.3/roles/wazuh/ansible-wazuh-manager/defaults/main.yml
+
+# Wazuh IP accessible from nodes running agents
+# Change network as needed
+wazuh_manager_ip: "{{ provision_oc_net_name | net_ip }}"
+
+# Single node setup
+single_node: true
+minimum_master_nodes: 1
+indexer_node_master: true
+
+# Configure hostnames for Wazuh indexer nodes
+# Example es1.example.com, es2.example.com
+domain_name: "dl"
+
+# Ansible control host certificate directory
+local_certs_path: "{{ playbook_dir }}/vars/certificates"
+
+# Ansible control host custom certificates directory
+local_custom_certs_path: "{{ playbook_dir }}/vars/custom_certificates"
+
+# Indexer variables
+indexer_node_name: "{{ inventory_hostname }}"
+
+# IP to which indexer will bind on host.
+indexer_network_host: "{{ provision_oc_net_name | net_ip }}"
+
+# Even in a single node setup this must be defined. If not defaults to 127.0.0.1
+indexer_cluster_nodes: 
+  - "{{ indexer_network_host }}"
+
+instances:
+  node1:
+    name: "{{ groups['wazuh-master'][0] }}"
+    ip: "{{ admin_oc_net_name | net_ip(groups['wazuh-master'][0]) }}"
+    role: indexer
+
+ansible_shell_allow_world_readable_temp: true
```

Comment:
Isn't that a bit insecure? At least deserves a comment why.

Reply:
It was introduced by this commit: https://github.com/wazuh/wazuh-ansible/commit/df92f85cbe761f28be9518806413867708e648fa 
Judging by commit message, it looks like it is leftover from past, carried forward in documentation and code. 
I checked removing  this with ubuntu installation and it went fine.